mycelo/genesis/genesis.go

```diff
@@ -0,0 +1,71 @@
+package genesis
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/core/types"
+	blscrypto "github.com/ethereum/go-ethereum/crypto/bls"
+	"github.com/ethereum/go-ethereum/mycelo/config"
+	"github.com/ethereum/go-ethereum/rlp"
+)
+
+// Keccak256 of "The Times 09/Apr/2020 With $2.3 Trillion Injection, Fed’s Plan Far Exceeds Its 2008 Rescue"
+var genesisMsgHash = common.HexToHash("ecc833a7747eaa8327335e8e0c6b6d8aa3a38d0063591e43ce116ccf5c89753e")
+
+// GenerateGenesis will create a new genesis block with full celo blockchain already configured
+func GenerateGenesis(env *config.Environment, contractsBuildPath string) (*core.Genesis, error) {
```

Comment:
if we change to GenesisAccount and GenesisConfig we don't need the Environment

Reply:
We do expand it in genesis/deployContext, but it probably can be removed.